src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -379,7 +379,17 @@ public String getActiveDeadlineSecondsStr() {
     }
 
     public Map<String, String> getLabelsMap() {
-        return ImmutableMap.of("jenkins/label", label == null ? DEFAULT_ID : label);
+        return ImmutableMap.of("jenkins/label", label == null ? DEFAULT_ID : sanitizeLabel(label));
+    }
+
+    private String sanitizeLabel(String input) {
```

Comment:
IIRC there was a similar utility method elsewhere in this plugin.

Reply:
Similar but not the same - https://github.com/jenkinsci/kubernetes-plugin/blob/3f91dcd194a4bbb2b45a21d07869928cec656cb0/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStepExecution.java#L166-L175